src/client/components/layout/Header.js

```diff
@@ -27,10 +31,12 @@ function Header() {
     <header>
       <AppBar position="static">
         <Toolbar>
-          <Logo className={classes.logo} />
-          <Typography variant="h6" color="inherit" className={classes.grow}>
-            {t('Explore')}
-          </Typography>
+          <a href="/" className={classes.link}>
```

Comment:
I don't know the convention, but maybe you can use `Link` from `react-router`.
And use  `HOME_ROUTE` for the link.

Reply:
I am not sure if the `Link` component works with static routing. When I tried it, I navigated to home but didn't get any collections (so I am not sure if it works with static routing and couldn't find information online)

Reply:
@juancarlosfarah  @hasanagh Indeed I have the same result. In fact the routing happens, but since the data are not fetched when mounting `Home` there are no collection (collections are only fetched when rendering from the server). 
- I don't really know the conventions, should we check somehow if the render is done from the server or the client to fetch the data from the component? 
- Maybe we can decide by looking at `collections` in the context, fetch the collections if the value is `null` (while fetching no collection would return  `[]`)
- Or just using a `a` is okay.  

But in any case you can change `HOME_ROUTE` in your `a` tag^^ This does work.

Reply:
Hey @pyphilia, I added `HOME_ROUTE`. As for the convention I am also not aware of it and checking how to deal with that.

Reply:
@pyphilia  check [this](https://phrase.com/blog/posts/localized-server-side-rendering-with-react/#The_Navigation_Bar)